way_to_home/place/urls.py

```diff
@@ -0,0 +1,10 @@
+"""The module that provide URL configuration for place app."""
+
+from django.urls import re_path
+
+from .views import PlaceView
+
+
+urlpatterns = [
+    re_path('place/(?P<place_id>\d+)?', PlaceView.as_view(), name='place'),
```

Comment:
why you use there re_path instead path?

Reply:
Because place_id is not required for post request. So instead to use for example `api/v1/place/1`, u will use just `api/v1/place`. I think it is impossible to do it with `path`. Or u know the solution?

Reply:
Ok, agree